torch/onnx/verification.py

```diff
@@ -816,7 +819,7 @@ def verify(
     ] = None,
     input_names: Optional[Sequence[str]] = None,
     output_names: Optional[Sequence[str]] = None,
-    training: torch.onnx.TrainingMode = torch.onnx.TrainingMode.EVAL,
+    training: _C_onnx.TrainingMode = _C_onnx.TrainingMode.EVAL,
     opset_version: Optional[int] = None,
```

Comment:
Having the same concern from https://github.com/pytorch/pytorch/pull/91925#discussion_r1065238517
Do we also have a default in here?

Reply:
Can we just make it required?

Reply:
> Having the same concern from [#91925 (comment)](https://github.com/pytorch/pytorch/pull/91925#discussion_r1065238517) Do we also need a default here?

It is mimicking export api, which accepts None as input. I agree perhaps it's best to just add the default at top level and make everywhere else required. On the other PR I wasn't sure if the fx-onnx internals intercepts the None and assigns default value somewhere.